Libraries/Image/ImageSourcePropType.js

```diff
@@ -43,6 +43,32 @@ const ImageURISourcePropType = PropTypes.shape({
    */
   body: PropTypes.string,
   /**
+   * `cache` determines how the requests handles potentially cached
+   * responses.
+   *
+   * - `default`: Use the native platforms default strategy. `useProtocolCachePolicy` on iOS.
+   *
+   * - `reload`: The data for the URL will be loaded from the originating source.
+   * No existing cache data should be used to satisfy a URL load request.
+   *
+   * - `force-cache`: The existing cached data will be used to satisfy the request,
+   * regardless of its age or expiration date. If there is no existing data in the cache
+   * corresponding the request, the data is loaded from the originating source.
```

Comment:
This terminology and behavior matches directly with the `fetch` API (please see earlier discussion in the thread). Even though it might be argued that the name is not descriptive enough, I believe there is a far bigger benefit sticking with already accepted and well used terminology than inventing our own.